Integrations/integration-Awake_Security.yml

```diff
@@ -0,0 +1,607 @@
+    import base64
+    import datetime
+    import re
+    import requests
+
+    params = demisto.params()
+
+    address = params["address"]
+    prefix = address + "/awakeapi/v1"
+
+    verify = params["verify"]
+
+    credentials = params["credentials"]
+    identifier = credentials["identifier"]
+    password = credentials["password"]
+    authTokenRequest = {
+      "loginUsername": identifier,
+      "loginPassword": password
+    }
+    authTokenResponse = requests.post(prefix + "/authtoken", json=authTokenRequest, verify=verify)
+    authToken = authTokenResponse.json()["token"]["value"]
+    headers = {
+        "Authentication": ("access " + authToken)
+    }
+
+    command = demisto.command()
+
+    args = demisto.args()
+
+    request = {}
+
+    # Convenient utility to marshal command arguments into the request body
+    def slurp(fields):
+        for field in fields:
+            if field in args:
+                request[field] = args[field]
+
+    # Render a subset of the fields of the Contents as a markdown table
+    def displayTable(contents, fields):
+        # We don't use a set() because we want to preserve field order
+        #
+        # The fields are ordered to put the most relevant information first
+        presentFields = []
+
+        # Omit table columns that are all empty
+        for content in contents:
+            for field in fields:
+                if field in content and content[field] and field not in presentFields:
+                    presentFields.append(field)
+
+        line0 = "| "
+        line1 = "| "
+        for field in presentFields:
+            # Translate camel-case field names to title-case space-separated words
+            tokens = re.findall("[a-zA-Z][A-Z]*[^A-Z]*", field)
+            name = " ".join(map(lambda token: token.title(), tokens))
+
+            line0 += name + " | "
+            line1 += "--- | "
+        line0 += "\n"
+        line1 += "\n"
+
+        body = ""
+        for content in contents:
+            body += "| "
+            for field in presentFields:
+                if field in content:
+                    value = json.dumps(content[field])
+                else:
+                    value = ""
+
+                body += value + " | "
+
+            body += "\n"
+
+        if presentFields:
+            return (line0 + line1 + body)
+        else:
+            return "Empty results"
+
+    def returnResults(contents, humanReadable, dbotScore):
+        machineReadable = {
+            "AwakeSecurity": contents,
+        }
+
+        if dbotScore is not None:
+            machineReadable["DBotScore"] = dbotScore
+
+        results = {
+            "Type": 1,
+            "ContentsFormat": "json",
+            "Contents": json.dumps(machineReadable),
+            "HumanReadable": humanReadable,
+            "ReadableContentsFormat": "markdown",
+        }
+
+        demisto.results(results)
+
+    def toDBotScore(indicator_type, percentile, lookup_key):
```

Comment:
maybe it's better instead of hardcoding the thresholds for the awake to dbot score, to add 2 params to the integration:
lower threshold
high threshold
we can give them any initial values we want.
and the dbotscoregiven can be according to them.
WDYT?